File: Semestre 2/Giluno/s3/mni_mvc1.rb
class Model
     def get_post
          {:title => "A test",:body =>"An example.."}
     end
end

class View
     def display(items)
          puts "Title:" + items[:title] + "\n" +'Body:' + items[:body]
     end
end     
class Controller
     def initialize
          @model = Model.new
          @view  = View.new
     end
     def main
          post = @model.get_post
          @view.display(post)
     end
end

mvc = Controller.new
mvc.main
     

File: Semestre 2/Giluno/s3/mni_mvc1.py
class Model:
     def get_post(self):
          return {"title":"A test","body":"An example.."}
     
class View:
     def display(self,items):
          print 'Title:',items['title'],'\n'+'Body:',items['body']
     
class Controller:
     def __init__(self):
          self.model = Model()
          self.view  = View()
     
     def main(self):
          post = self.model.get_post()
          self.view.display(post)
     
mvc = Controller()
mvc.main()
     